app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for bro in friends:
+        try:
+            cafe.visit_cafe(bro)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+    if masks_to_buy > 0:
```

Comment:
You can simply check `if masks_to_buy:`, if `masks_to_buy` equal to 0 it will be False, else True. So no sense for this comparison